packages/badge/README.md

```diff
@@ -0,0 +1,49 @@
+## Description
+
+Badges show a small amount of color-categorized metadata. They're ideal for getting a user's attention.
+
+### Usage
+
+[![See it on NPM!](https://img.shields.io/npm/v/@spectrum-web-components/badge?style=for-the-badge)](https://www.npmjs.com/package/@spectrum-web-components/badge)
+[![How big is this package in your project?](https://img.shields.io/bundlephobia/minzip/@spectrum-web-components/badge?style=for-the-badge)](https://bundlephobia.com/result?p=@spectrum-web-components/badge)
+
+```
+yarn add @spectrum-web-components/badge
+```
+
+Import the side effectful registration of `<sp-badge>` via:
+
+```
+import '@spectrum-web-components/badge/sp-badge.js';
+```
+
+When looking to leverage the `Badge` base class as a type and/or for extension purposes, do so via:
+
+```
+import { Badge } from '@spectrum-web-components/badge';
+```
+
+## Example
+
+Badges should always have a label for accessibility and clear comprehension.
```

Comment:
Could a badge be simply an icon? In which case the Badge or the Icon would need `aria-label` or similar to also prove the accessibility of the delivery...

Reply:
The [spec](https://spectrum.adobe.com/page/badge/#Label) says a badge "should always have a label," but also demonstrates icon-only badges.

I think specifying aria should probably be done by the user, if the user opts to omit a label - right? eg:

```html
<sp-badge aria-label="filters enabled">
  <sp-icon-checkmark-circle slot="icon"></sp-icon-checkmark-circle>
</sp-badge>
```

Reply:
Yes. `sp-icon-*` also accept a `label` attribute.